support.sh

```diff
@@ -18,20 +18,69 @@ function die {
 type -P ${DOCKER} > /dev/null || die "This tool requires the docker binary"
 type -P ${NSENTER} > /dev/null || die "This tool requires nsenter"
 type -P ${BRIDGE} > /dev/null || die "This tool requires bridge"
-type -P ${BRCTL} > /dev/null || die "This tool requires brctl"
 type -P ${IPTABLES} > /dev/null || die "This tool requires iptables"
+type -P ${IPVSADM} > /dev/null || die "This tool requires ipvsadm"
+type -P ${IP} > /dev/null || die "This tool requires ip"
 
-echo "iptables configuration"
-${IPTABLES} -w1 -n -v -L -t filter
-${IPTABLES} -w1 -n -v -L -t nat
+echo "Host iptables configuration"
+printf "\n# filter table\n"
+${IPTABLES} -w1 -n -v -L -t filter | grep -v '^$'
+printf "\n# nat table\n"
+${IPTABLES} -w1 -n -v -L -t nat | grep -v '^$'
+printf "\n# mangle table\n"
+${IPTABLES} -w1 -n -v -L -t mangle | grep -v '^$'
+printf "\n"
+
+echo "Host addresses and routes"
+${IP} -o -4 address show
+${IP} -4 route show
 echo ""
 
 echo "Overlay network configuration"
-for networkID in $(${DOCKER} network ls --filter driver=overlay -q) ; do
-    echo "Network ${networkID}"
-    nspath=(${NSDIR}/*-$(echo ${networkID}| cut -c1-10))
-    ${DOCKER} network inspect -v ${networkID}
-    ${NSENTER} --net=${nspath[0]} ${BRIDGE} fdb show ${BRIDGEIF}
-    ${NSENTER} --net=${nspath[0]} ${BRCTL} showmacs ${BRIDGEIF}
-    echo ""
+for networkID in $(${DOCKER} network ls --filter driver=overlay -q) "ingress_sbox"; do
+    echo "nnn Network ${networkID}"
+    if [ "${networkID}" != "ingress_sbox" ]; then
+        nspath=(${NSDIR}/*-${networkID:0:10})
```

Comment:
maybe not a problem, but if an overlay network exists but is unused on the node where the tool is used, it will throw errors:

```
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
nsenter: cannot open /var/run/docker/netns/1-r5isn62y0l: No such file or directory
```

Reply:
This was already the behaviour - we can clean this up if we like

Reply:
We can emit a message like "network unused on this node"

Reply:
not a big deal, I'm more worried about the clash of the condition, the network is not used so the namespace is not there that is expected, or the namespace was supposed to be there but instead is not and this is a bug.